modules/src/main/java/eu/itesla_project/modules/online/OnlineWorkflowParameters.java

```diff
@@ -114,6 +117,7 @@ public OnlineWorkflowParameters(DateTime baseCaseDate, int states, Interval hist
         this.limitReduction = limitReduction;
         this.handleViolationsInN = handleViolationsInN;
         this.constraintMargin = constraintMargin;
+        this.caseFile = caseFile;
```

Comment:
The input parameters check should be more complex: (case-date, countries, case-type) are all not null xor case-file is not null.